cwms-data-api/src/main/java/cwms/cda/api/LocationController.java

```diff
@@ -280,6 +281,13 @@ public void getOne(@NotNull Context ctx, @NotNull String name) {
                         @OpenApiContent(from = Location.class, type = Formats.XML)
                     },
                     required = true),
```

Comment:
@MikeNeilson - as a general question, does it make more sense to you to not have a fail-if-exists overwrite for 'create'? Create, at least to me, implies is does not yet exist - and if it does it should fail with an error. Whereas, 'update' should be used if it does exist. I just question the need for this fail-if-exists query parameter.

Reply:
in general, now, but it's a fundamental part of our our concepts have worked in the past that just isn't mapping to the HTTP verbs as well as we'd like.